test/phabricator_tests/test_phabricator.py

```diff
@@ -0,0 +1,658 @@
+                                        mock_dfferential_query,
+                                        mock_generate_phids,
+                                        mock_Phabricator):
+        """
+        Tests request_reviews with user names specified
+        """
+        fake_phab = MagicMock()
+        mock_generate_phids.return_value = ([], 'PHID-USER-xxxxxxxxxxxxxxxxxxxx')
+        mock_dfferential_query.return_value = 'mock_reviews'
+        mock_Phabricator.return_value = fake_phab
+        mock_get_reviews.return_value = '1'
+
+        response = PhabricatorService().request_reviews(
+            host="https://www.dummy.com",
+            token="dummy_token",
+            user_names=None,
+            state_=None,
+            value=None,
+            duration=None,
+            show_last_comment=None)
+
+        mock_dfferential_query.assert_called_with(
+            status='status-open',
+            responsibleUsers=[],
+            phab=fake_phab
+        )
+
+        mock_Phabricator.assert_called_with(
+            host="https://www.dummy.com/api/",
+            token="dummy_token"
+        )
+
+        self.assertEqual(['1'], response)
+
+    @patch(PATH + 'Phabricator')
+    @patch(PATH + 'PhabricatorService.generate_phids')
+    @patch(PATH + 'PhabricatorService.differential_query')
+    @patch(PATH + 'PhabricatorService.get_reviews')
+    def test_request_reviews_user_names(self, mock_get_reviews,
+                                        mock_dfferential_query,
+                                        mock_generate_phids,
+                                        mock_Phabricator):
+        """
+        Tests request_reviews with user names specified
+        """
+        fake_phab = MagicMock()
+        mock_generate_phids.return_value = ([], 'PHID-USER-xxxxxxxxxxxxxxxxxxxx')
+        mock_dfferential_query.return_value = 'mock_reviews'
+        mock_Phabricator.return_value = fake_phab
+        mock_get_reviews.return_value = '1'
+
+        response = PhabricatorService().request_reviews(
+            host="https://www.dummy.com",
+            token="dummy_token",
+            user_names='test_user',
+            state_=None,
+            value=None,
+            duration=None,
+            show_last_comment=None)
+
+        mock_generate_phids.assert_called_with(
+            'test_user', fake_phab
+        )
+
+        mock_dfferential_query.assert_called_with(
+            status='status-open',
+            responsibleUsers=[],
+            phab=fake_phab
+        )
+
+        mock_Phabricator.assert_called_with(
+            host="https://www.dummy.com/api/",
+            token="dummy_token"
+        )
+
+        self.assertEqual(['1'], response)
+
+    @patch(PATH + 'Phabricator')
+    @patch(PATH + 'PhabricatorService.generate_phids')
+    @patch(PATH + 'PhabricatorService.differential_query')
+    @patch(PATH + 'PhabricatorService.get_reviews')
+    def test_request_reviews_no_user_names(self, mock_get_reviews,
```

Comment:
Ha! Found it!

This is actually the same test as the first `test_request_reviews_user_names` (I am saying first because you unintentionally have created two tests with the same name, see comment above). I think you wanted to have this

```python
mock_get_reviews.return_value = None
```
and this
```python
response = PhabricatorService().request_reviews(
    ...
    user_names='test_user',
    ...
```